tracing-log/src/lib.rs

```diff
@@ -221,6 +221,27 @@ impl AsTrace for log::Level {
 
 /// A simple "logger" that converts all log records into `tracing` `Event`s,
 /// with an optional level filter.
+///
+/// Be careful to set appropriate log levels:
+///
+/// * `LogTracer`'s default filter level is `log::LevelFilter::Info`.
+/// * `log`'s default max level is `log::LevelFilter::Off` (logging disabled).
+///
+/// For example, to get trace logs:
+///
+/// ```rust
+/// # use tracing_log::LogTracer;
+/// # use std::error::Error;
+/// # use log::trace;
+/// # fn main() -> Result<(), Box<Error>> {
+/// let logger = LogTracer::with_filter(log::LevelFilter::Trace);
+/// log::set_boxed_logger(Box::new(logger))?;
+/// log::set_max_level(log::LevelFilter::Trace);
+///
+/// trace!("an example trace log");
```

Comment:
Perhaps the macro should be used namespaced here, so that it's clear it's a `log` macro rather than a `tracing` macro?
```suggestion
/// log::trace!("an example trace log");
```

Reply:
indeed (I actually did it with the functions to make it clear they come from `log`, and skipped the most important one!)